source/includes/api/performance/_self_claims.md

```diff
@@ -1,5 +1,92 @@
 ## Self Claims
 
+### Create a Self Claim
+
+Endpoint to create a self claim against a specific self-claim promotion.
+
+#### HTTP Request
+
+`POST /api/v2/performance/promotions/{promotion_id}/self_claims`
+
+> Header:
+
+``` http
+POST /api/v2/performance/promotions/1/self_claims HTTP/1.1
+Authorization: Token token=key:secret
+Content-Type: application/json
+```
+
+> Body:
+
+```json
+{
+  "user_id": 1,
+  "sale_date": "2022-01-29",
+  "product_or_activity_ref": "p1",
+  "quantity": 150,
+  "dynamic question free text?": "Free text answer",
+  "dynamic question date select": "2017-06-21",
+  "dynamic question scanner": "abcdefg",
+  "dynamic question list": "Choice A, Choice B",
+  "dynamic question file upload": "https://invoicefile/12353.doc"
```

Comment:
Have this in the same format as the response - i.e. an object/hash of `{ question_id => answer }`, and nest under an `answer` key?

Relying on question name seems unreliable, since someone could just change the name in the admin UI without thinking and then break integrations/lambdas etc.